client/src/components/pages/Profile/index.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { Tabs, Icon } from "antd";
+import PropTypes from "prop-types";
+
+import PersonalInfoForm from "components/utils/ProfilePersonal";
+import BusinessForm from "components/utils/businessForm";
+import ProfilePic from "components/pages/Profile/ProfilePic";
+import "./style.css";
+
+const { TabPane } = Tabs;
+
+export default function ProfilePage({
+  personal = undefined,
+  business = undefined,
+  className = ""
+}) {
+  return (
+    <div className={className}>
+      <ProfilePic className="profile-page--pic" />
+      <Tabs defaultActiveKey="1" animated={true} className="profile-page-tabs">
+        <TabPane
+          tab={
+            <span>
+              <Icon type="user" />
+              Personal Info
+            </span>
+          }
+          key="1"
+        >
+          <PersonalInfoForm personal={personal} />
+        </TabPane>
+        <TabPane
+          tab={
+            <span>
+              <Icon type="project" />
+              Business
+            </span>
+          }
+          key="2"
+        >
+          <BusinessForm business={business} />
+        </TabPane>
+      </Tabs>
+    </div>
+  );
+}
+
+ProfilePage.propTypes = {
+  personal: PropTypes.object.isRequired,
+  business: PropTypes.object.isRequired,
```

Comment:
@ahmedisam99 It looks like you have `isRequired` for these props, but you provide default values for them up on lines 13 and 14.  It might be less confusing to others who are using this component if you either make them required, without defaults, or keep the defaults and remove the isRequired.  That'll help people to know how they're supposed to use this component.

Reply:
I thought although it is a required prop it would be better to have a fall back value that won't crash the app if anything goes wrong

Reply:
Ah ok, sounds good